client/src/app/shared/shared-main/misc/top-menu-dropdown.component.html

```diff
@@ -4,12 +4,8 @@
     <a *ngIf="menuEntry.routerLink && isDisplayed(menuEntry)" [routerLink]="menuEntry.routerLink" routerLinkActive="active" class="title-page title-page-settings">{{ menuEntry.label }}</a>
 
     <div *ngIf="!menuEntry.routerLink && isDisplayed(menuEntry)" ngbDropdown class="parent-entry"
-      #dropdown="ngbDropdown" autoClose="outside">
+      #dropdown="ngbDropdown" autoClose="true">
       <span
-        *ngIf="isInSmallView"
```

Comment:
Is this deletion a mistake?

Reply:
I put it back in https://github.com/Chocobozzz/PeerTube/commit/805d6a354bd51fccd434c0c1b1df69c81bb90dd4 (we can revert if it was not a mistake) because it causes duplicate entries in the admin menu

Reply:
Yes, thanks. Sorry, it was a mistake, without it would indeed duplicate the entry with one for the modal view and another for the dropdown view.